src/platform/item/linked-wearable-mappings.ts

```diff
@@ -173,3 +173,103 @@ export namespace Mapping {
     RangeMapping._fromLessThanOrEqualTo
   ])
 }
+
+export type Network = 'mainnet' | 'matic' | 'sepolia' | 'amoy'
+export type ContractAddress = string
+
+export type Mappings = Partial<Record<Network, Record<ContractAddress, Mapping[]>>>
+
+export type MappingsHelper = {
+  addMapping(network: Network, contractAddress: ContractAddress, mapping: Mapping): void
+  includesNft(network: Network, contractAddress: ContractAddress, tokenId: string): boolean
+}
+
+export function createMappingsHelper(): MappingsHelper {
```

Comment:
Can we add a way to initialize this mappings helper using a Mappings object? It would be really helpful to work with an already created mapping as well.